packages/rx_bloc_list/lib/src/extensions/identifiable_pair_extensions.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:rxdart/rxdart.dart';
+
+import '../../models.dart';
+
+extension ModelManageEventsPair<E extends Identifiable>
+    on Stream<IdentifiablePair<E>> {
+  Stream<ManagedList<E>> withLatestFromIdentifiablePairList(
+    Stream<List<E>> list, {
+    required OperationCallback<E> operationCallback,
+  }) =>
+      _withLatestFromListPair(list).flatMap((tuple) async* {
+        switch (await operationCallback(tuple.pair.updated)) {
+          case ManageOperation.merge:
+            yield ManagedList(
+              tuple.list.mergeWithList([tuple.pair.updated]),
+              operation: ManageOperation.merge,
+              identifiable: tuple.pair.old,
+              identifiableInList: tuple.pair.updated,
```

Comment:
Hmmm, can we think of better naming of those props. `identifiable`, `identifiableInList`

Reply:
Use the PairClass instead of these

Reply:
I have replaced both of the `identifiable` and `identifiableInList` with the pair object